edb/pgsql/delta.py

```diff
@@ -3739,19 +3739,39 @@ def apply(
             # the indexes are dropped automatically in this case.
```

Comment:
Huh, I'm surprised we have this optimization for indexes. We mostly don't for anything else. (We delete columns one at a time before deleting the table. Maybe we should do better though)